wca-guidelines.md

```diff
@@ -144,6 +144,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-11><incidents><incidents> Article 11: Incidents
 
+- 11d1+) [EXAMPLE] Examples of incidents where it is appropriate to waive a penalty: touching a camera during inspection (see [Regulation 2i2b](regulations:regulation:2i2b)), lightly touching a puzzle at the end of the attempt (see [Regulation A6e](regulations:regulation:A6e)).
```

Comment:
Why `lightly` and not `slightly`? Is that because the one you use is American English and the other is British?

Reply:
Perhaps. I think "lightly" is definitely more idiomatic.

Reply:
"lightly touch" seems to have about 1x as many Google results as "slightly touch", so I think I'd like to stay with that.